app/src/main/java/com/github/se/travelpouch/model/profile/ProfileRepository.kt

```diff
@@ -55,4 +55,11 @@ interface ProfileRepository {
       onSuccess: (Profile) -> Unit,
       onFailure: (Exception) -> Unit
   )
+
+  fun removeFriend(
```

Comment:
Please add minimal documentation to this function

Reply:
Done